luda-editor/luda-editor-client/src/editor/mod.rs

```diff
@@ -24,10 +28,13 @@ struct Editor {
     job: Option<Job>,
     timeline: Timeline,
     clip_editor: ClipEditor,
+    sequence_list_view: SequenceListView,
     playback_time: chrono::Duration,
     socket: luda_editor_rpc::Socket,
     screen_wh: namui::Wh<f32>,
     image_filename_objects: Vec<ImageFilenameObject>,
+    sequence_load_state_map: SequenceLoadStateMap,
```

Comment:
Editor have `sequence_load_state_map`: `HashMap<String: path, SequenceLoadState>`;

Reply:
`Editor` itself has the context of single sequence editing, so I think it's not a good idea to put multiple sequence's loading information or page information inside of `Editor`. 
Would you like to make a upper object to handle that?

Reply:
Aaah. you mean `Main` in `main.rs` right?
I was mistaken for `Editor` to be top object.
I will fix it

Reply:
I don't think main.rs is right place. New concept like App needed